pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -1139,6 +1139,9 @@ func newRoleBinding(namespace, name string) *v1.RoleBinding {
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: namespace,
 			Name:      name,
+			Labels: map[string]string{
+				toolchainv1alpha1.ProviderLabelKey: toolchainv1alpha1.ProviderLabelValue,
```

Comment:
May I know why did you add the label here, and not in the `newRole` function? In both cases, the resource is created as part of the test setup.

Reply:
fixed as part of: ad15c624078af2b4d102df7ed1304e6483044976